CHANGELOG.md

```diff
@@ -3354,3 +3355,4 @@
 [@JacobEvelyn]: https://github.com/JacobEvelyn
 [@shanecav84]: https://github.com/shanecav84
 [@thomasbrus]: https://github.com/thomasbrus
+[@huyvohcmc]: https://github.com/huyvohcmc
```

Comment:
Good catch! but it isn't a feature change to end users, addition to CHANGELOG.md is unnecessary.

Reply:
I'll remove it.